model/src/test/kotlin/licenses/LicenseInfoResolverTest.kt

```diff
@@ -672,9 +709,14 @@ fun containOnlyLicenseSources(vararg licenseSources: LicenseSource): Matcher<Res
         )
     }
 
-fun containLicenseExpressionsExactly(vararg expression: SpdxExpression?): Matcher<ResolvedLicenseInfo?> =
+fun containOriginalLicenseExpressionsBySourceExactly(
+    sources: Set<LicenseSource>,
+    vararg expression: SpdxExpression?
+): Matcher<ResolvedLicenseInfo?> =
     neverNullMatcher { resolvedLicenseInfo ->
-        val actualExpressions = resolvedLicenseInfo.licenseExpressions.toSet()
+        val licensesBySource = resolvedLicenseInfo.licenses.filter { it.sources.containsAll(sources) }
```

Comment:
You could use a `LicenseView` as parameter for the function and then use this instead:
`val licensesBySource = view.filter(resolvedLicenseInfo)`

Reply:
We removed the sources in this test function altogether, so we don't need the filter anymore.

Reply:
We added the sources again, but don't think the filtering done with the licenseView would improve this code.
Please have a look at the changed code.